etc/kayobe/kolla.yml

```diff
@@ -447,6 +447,79 @@
 #kolla_enable_watcher:
 #kolla_enable_zun:
 
+###############################################################################
```

Comment:
I suppose this should go into SKC.

Reply:
Good call, proposed here: https://github.com/stackhpc/stackhpc-kayobe-config/pull/778